tasks/codex/edm.py

```diff
@@ -0,0 +1,525 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is full of non-tested lines.  To add them to the automated testing procedure, add new lines for each of the new EDM tasks to `scripts/tests/run_tasks_for_coverage.sh`

Reply:
That is very much intentional. This is the util that will be used by @naknomum for moving data from EDM to houston and requires EDM and it's houston extension started manually. These will never be enabled on the github environment so I'd argue that there is no reason to add this to the coverage stats and a very significant numbe of resons not to.

Reply:
Although I think it is a fair comment to ask @naknomum to note in here the steps he needs to do to get this to run.... just in case any of the rest of us ever need to impersonate him :)